gcc/rust/expand/rust-macro-expand.cc

```diff
@@ -3125,7 +3156,7 @@ MacroExpander::expand_decl_macro (Location invoc_locus,
       RichLocation r (invoc_locus);
       r.add_range (rules_def.get_locus ());
       rust_error_at (r, "Failed to match any rule within macro");
-      return AST::ASTFragment::create_empty ();
+      return AST::ASTFragment::create_error ();
```

Comment:
do we still need to return the ASTFragments to be part of the MacroInvocation since it seems we now issue a fatal_error on that?